hypertrace-trace-enricher/hypertrace-trace-enricher-impl/src/test/java/org/hypertrace/traceenricher/enrichment/enrichers/resolver/backend/RabbitMqBackendResolverTest.java

```diff
@@ -0,0 +1,82 @@
+package org.hypertrace.traceenricher.enrichment.enrichers.resolver.backend;
+
+import static org.mockito.Mockito.mock;
+
+import com.google.common.base.Joiner;
+import java.nio.ByteBuffer;
+import java.util.Arrays;
+import java.util.Map;
+import org.hypertrace.core.datamodel.AttributeValue;
+import org.hypertrace.core.datamodel.Attributes;
+import org.hypertrace.core.datamodel.Event;
+import org.hypertrace.core.datamodel.EventRef;
+import org.hypertrace.core.datamodel.EventRefType;
+import org.hypertrace.core.datamodel.MetricValue;
+import org.hypertrace.core.datamodel.Metrics;
+import org.hypertrace.core.datamodel.shared.StructuredTraceGraph;
+import org.hypertrace.entity.constants.v1.K8sEntityAttribute;
+import org.hypertrace.entity.data.service.client.EntityDataServiceClient;
+import org.hypertrace.entity.data.service.v1.Entity;
+import org.hypertrace.traceenricher.enrichment.enrichers.resolver.FQNResolver;
+import org.hypertrace.traceenricher.util.Constants;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Unit test for {@link RabbitMqBackendResolver}
+ */
+public class RabbitMqBackendResolverTest {
+
+  private static final String NAMESPACE = "traceable";
+  private static final String CLUSTER = "devcluster";
+
+  private EntityDataServiceClient edsClient;
+
+  private RabbitMqBackendResolver rabbitMqBackendResolver;
+  private StructuredTraceGraph structuredTraceGraph;
+
+  @BeforeEach
+  public void setup() {
+    rabbitMqBackendResolver = new RabbitMqBackendResolver(new FQNResolver(edsClient));
+    edsClient = mock(EntityDataServiceClient.class);
+    structuredTraceGraph = mock(StructuredTraceGraph.class);
+  }
+
+  @Test
+  public void testEventResolution() {
+    String routingKey = "routingkey";
+    Entity entity = rabbitMqBackendResolver.resolveEntity(getRabbitMqEvent(routingKey), structuredTraceGraph).get();
+    Assertions.assertEquals(Joiner.on(".").join(routingKey, NAMESPACE, CLUSTER),
```

Comment:
Let's have a expected value instead of  `Joiner.on(".").join(routingKey, NAMESPACE, CLUSTER)`

Reply:
But that would mean hardcoding values, which makes it difficult to maintain.